trin-execution/src/cli.rs

```diff
@@ -54,12 +56,16 @@ pub enum TrinExecutionSubCommands {
 
 #[derive(Args, Debug, Default, Clone, PartialEq)]
 pub struct ImportStateConfig {
+    #[arg(long, help = "The directory where to import the snapshot data.")]
+    pub data_dir: Option<PathBuf>,
```

Comment:
Shouldn't we just use the `data_dir` from `TrinExecutionConfig`. As with this the user can input 2 different data_dirs potentially and the CLI won't complain, but then the user will be confused. So I think we should use the one in `TrinExecutionConfig`